app/controllers/pages/questions_controller.rb

```diff
@@ -0,0 +1,83 @@
+class Pages::QuestionsController < PagesController
```

Comment:
At the moment, its following what all the other add/edit pages controllers are doing but potentially it could inherit from a `BaseController` instead. Theres still more work to do on this but will keep this comment in mind as the refactoring goes along.